### DISCUSSION TITLE
How to increase 'minmax' FontSize?

### DISCUSSION THREAD

**Discussion by Gbowker** [mtex-toolbox/mtex#542] [Ask Anything] [answered]
Just a simple question - how to increase the 'minmax' text for a pole figure so it is more readable?

**Comment by kilir** [comment, marked as answer]
Hi,
for example in `mtex_settings` or directly with `setMTEXpref('FontSize',fontSize);`.
If you just want to change those numbers e.g.:
```matlab
plot(vector3d.rand(10),'contourf','minmax')
f = gcm;
for i=1:length(f.children)
ax = getappdata(f.children(i),'sphericalPlot');
ax.TL.FontSize=36; % TL= top left
ax.BL.FontSize=5; % TL= bottom left
end 
```
Cheers,
Rüdiger

**Reply by Gbowker** [reply]
Thanks very much. 👍
